### DISCUSSION TITLE
Arena Sport 1 CRO

### DISCUSSION THREAD

**Discussion by TurboPing** [iptv-org/community#876] [Q&A]
### Channel ID

ArenaSport1.hr

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://cloudcdn.tk:80/Visarhem/lQzhj3068/19004
<img width="786" alt="image" src="https://user-images.githubusercontent.com/7253922/234043790-87232926-7ed0-455c-90b6-3f63e18a8eaf.png">

**Comment by kaikyoofers** [comment]
> http://cloudcdn.tk:80/Visarhem/lQzhj3068/19004 <img alt="image" width="786" src="https://user-images.githubusercontent.com/7253922/234043790-87232926-7ed0-455c-90b6-3f63e18a8eaf.png">

Nope, xtream link @freearhey

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19832
